src/components/StyledButtons.js

```diff
@@ -16,6 +16,7 @@ const StyledButton = styled.button`
 	${({variant = 'goback'}) =>
 		variant === 'goback' &&
 		css`
+			position: fixed;
 			padding-left: 20px;
 		`}
 	${({variant = 'plus'}) =>
```

Comment:
Please check my comments in your previous PR... 

```suggestion
	${({variant}) =>
		variant === 'goback' &&
		css`
			position: fixed;
			padding-left: 20px;
		`}
	${({variant}) =>
```

Reply:
will do thank you